iroha-cpp/tx-example.cpp

```diff
@@ -0,0 +1,159 @@
+#include <boost/filesystem.hpp>
+#include <iostream>
+
+#include "crypto/keys_manager_impl.hpp"
+#include "logger/logger_manager.hpp"
+#include "model/converters/pb_common.hpp"
+
+#include "services/Query.hpp"
+#include "services/Status.hpp"
+#include "services/Tx.hpp"
+#include "GrpcResponseHandler.hpp"
+
+
+namespace fs = boost::filesystem;
+
+
+auto log_manager = std::make_shared<logger::LoggerManagerTree>(logger::LoggerConfig{logger::LogLevel::kInfo, logger::getDefaultLogPatterns()})->getChild("CLI");
+const auto response_handler_log_manager = log_manager->getChild("ResponseHandler");
+const auto pb_qry_factory_log = log_manager->getChild("PbQueryFactory")->getLogger();
+
+
+void verifyPath(const fs::path& path, const logger::LoggerPtr& logger)
+{
+    if (not fs::exists(path))
+    {
+        const auto error_message = "Path {} not found." + path.string();
+        logger->error(error_message);
+        throw error_message;
+    }
+}
+
+
+void verifyKepair(const iroha::expected::Result<shared_model::crypto::Keypair, std::string>& keypair,
+                  const logger::LoggerPtr& logger,
+                  const fs::path& path,
+                  const std::string& account_name)
+{
+    if (auto e = iroha::expected::resultToOptionalError(keypair))
+    {
+        const auto error_message = "Keypair error: "
+                + e.value()
+                + ".\nKeypair path: '"
+                + path.string()
+                + "', name: "
+                + account_name
+                + ".\n";
+        logger->error(error_message);
+        throw error_message;
+    }
+}
+
+
+auto generateKeypair(const std::string& account_name,
```

Comment:
IMO in C++ it would be better to extract methods used in examples to separate file, which would be included from all examples.

I know that in iroha-python all examples are independent, but in compiled language (C++) it is not problem when we have more files.

Reply:
Outdated comment.